packages/e2e-tests/fixtures/blocks/core__file__new-window.html

```diff
@@ -1,3 +1,3 @@
 <!-- wp:file {"href":"http://localhost:8888/wp-content/uploads/2018/05/keycodes.js","showDownloadButton":true} -->
-<div class="wp-block-file"><a href="http://localhost:8888/wp-content/uploads/2018/05/keycodes.js" target="_blank" rel="noreferrer noopener">6546</a><a href="http://localhost:8888/wp-content/uploads/2018/05/keycodes.js" class="wp-block-file__button" download>Download</a></div>
```

Comment:
As a deprecation is needed, the practice here is to keep the old markup in a fixture called ` packages/e2e-tests/fixtures/blocks/core__file__deprecated-1.html` which will add test coverage ensuring the deprecation works. The new fixture can be added as well alongside it.

Reply:
Ok, will do.

Reply:
Hey @talldan - finally getting back to this :)

Since there is now already a `core__file__new-window__deprecated-1.html` I assume I need to create `core__file__new-window__deprecated-2.html`? do I then leave the original and first deprecation untouched?